helper-cli/src/main/kotlin/commands/ImportCopyrightGarbageCommand.kt

```diff
@@ -58,13 +58,12 @@ internal class ImportCopyrightGarbageCommand : CliktCommand(
 
     override fun run() {
         val entriesToImport = inputCopyrightGarbageFile
-            .expandTilde()
             .readText()
             .lines()
             .filterNot { it.isBlank() }
 
         val existingCopyrightGarbage = if (outputCopyrightGarbageFile.isFile) {
-            outputCopyrightGarbageFile.expandTilde().readValue<CopyrightGarbage>().items
+            outputCopyrightGarbageFile.readValue<CopyrightGarbage>().items
```

Comment:
commit-message: 

"expandTilde() is already called by the clikt option conversion."  IMO is misleading because the conversion calls
`String.expandTilde()` while the call to `File.expandTile()` is removed, and the semantics is a bit different.

Reply:
I've clarified that.

Reply:
IIRC correctly one of the differences was that the file one additional converts relative to absolute paths.
Maybe that was worth mentioning.

Reply:
Reworded the commit message yet again accordingly.